### DISCUSSION TITLE
Previous objects could be replaced when a new one is placed on the same tick to match stable.

### DISCUSSION THREAD

**Discussion by gustarmartins** [ppy/osu#20853] [Ideas]
Currently, when you place an object on the same tick as the other, it won't automatically erase the previous object.

**Comment by peppy** [comment]
I'm not too sure about this. I feel like in some ways the lazer behaviour is better?

Consider this case, where a user copies some objects and then pastes but the paste (unintentionally?) overwrites existing objects:


https://user-images.githubusercontent.com/191335/197462696-d963acd8-24b9-412f-84aa-c3781fe5fa5a.mp4


https://user-images.githubusercontent.com/191335/197463058-17a2a1a2-e5d4-4942-9768-09eb49a7ba1e.mp4

Interested in more opinions on this. If the first is preferable maybe we need a "Paste with overwrite" option as a non-default? I feel like the latter is the more user-friendly one.
